code/datums/wounds/robotic/robotic_pierce.dm

```diff
@@ -0,0 +1,166 @@
+#define ELECTRICAL_DAMAGE_REPAIR_WELD_BASE_DELAY 3 SECONDS
+#define ELECTRICAL_DAMAGE_REPLACE_METALS_BASE_DELAY 4 SECONDS
+
+/datum/wound/electrical_damage/pierce
+	wound_type = WOUND_PIERCE
+	wound_series = WOUND_SERIES_WIRE_PIERCE_ELECTRICAL_DAMAGE
+
+/datum/wound_pregen_data/electrical_damage/pierce
+	abstract = TRUE
+
+/datum/wound/burn/electrical_damage/pierce/get_limb_examine_description()
+	return span_warning("The metal on this limb is pierced open.")
+
+/datum/wound/electrical_damage/pierce/moderate
+	name = "Punctured Capacitor"
+	desc = "A major capacitor has been broken open, causing slow but noticable electrical damage."
+	occur_text = "shoots out a short stream of sparks"
+	examine_desc = "is shuddering gently, movements a little weak"
+	treat_text = "Replacing of damaged wiring, though repairs via wirecutting instruments or sutures may suffice, albiet at limited efficiency. In case of emergency, \
+				subject may be subjected to high temperatures to allow solder to reset."
```

Comment:
```suggestion
	treat_text = "Replacing of damaged wiring, though repairs via wirecutting instruments or sutures may suffice albeit at limited efficiency. In case of emergency, \
		subject may be subjected to high temperatures to allow solder to reset."
```